README.md

```diff
@@ -10,6 +10,24 @@
 [![Gitter](https://badges.gitter.im/jenkinsci/configuration-as-code-plugin.svg)](https://gitter.im/jenkinsci/configuration-as-code-plugin)
 
 <img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
+## Documentation
+
+<a name="handling-secrets"></a>
+
+You can find more documentation about JCasC here:
+
+- [Handling Secrets](./docs/features/secrets.adoc)
+- [Exporting configurations](./docs/features/configExport.md)
+- [Validating configurations](./docs/features/jsonSchema.md)
+- [Triggering Configuration Reload](./docs/features/configurationReload.md)
+
+The configuration file format depends on the version of jenkins-core and installed plugins.
+Documentation is generated from a live instance, as well as a JSON schema you can use to validate configuration file
+with your favourite YAML tools.
+
+The JSON Schema documentation can be found [here](./docs/features/jsonSchema.md).
+
+**TODO**: Provide a Dockerfile to generate documentation from specified jenkins-core release and plugins.
```

Comment:
We can probably remove this or at least comment it out, users don’t need to see it.

Since it’s being made more prominent it’s worth addressing now, (I’ve never noticed it before)

Reply:
done